types/jest/index.d.ts

```diff
@@ -1,4 +1,4 @@
-// Type definitions for Jest 29.4
+// Type definitions for Jest 29.5
```

Comment:
Why version bump? This is feature was added in [Jest v29.4](https://github.com/facebook/jest/releases/tag/v29.4.0).

```suggestion
// Type definitions for Jest 29.4
```

Reply:
Reverted this change, thanks 😉